charts/mysql/ticket-template.md

```diff
@@ -0,0 +1,119 @@
+This template is designed to contact with AppsCode with troubleshooting and support in general. You will find all the KubeDB Managed database follows similar pattern. To troubleshoot and find what to do let's get started with the database phase.
+
+Let's say you have MySQL database in namespace demo.Please refer to [KubeDB Docs](https://kubedb.com/docs/v2023.01.31/guides/mysql/) for more about KubeDB.
```

Comment:
Is this link static to release v2023.01.31? If yes, it should be dynamic or we can just forward it to kubedb.com/docs

Reply:
I'm not aware how to do it . any idea?

Reply:
https://kubedb.com/docs/latest/guides/redis/ seems to work

Reply:
@faem vai

Reply:
yap. use `latest` instead of specific version.